.github/actions/code-checks/action.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Code Checks'
+description: 'Run tests, linting, and other common jobs'
+runs:
+  using: 'composite'
```

Comment:
What does this do?

Reply:
This is a way of bundling a set of shell commands in a single action. Useful if you want to re-use a bunch of lines across multiple jobs / workflows.